worker_node/job/base.js

```diff
@@ -0,0 +1,29 @@
+
+class Base {
+  #config;
+  #output;
+
+  constructor(config) {
+    this.#config = config;
```

Comment:
Input "config" requires validation.
You also need to know what values will be in "config".

Reply:
Done. Add validation logic to base